tests/lean/infoTree.lean.expected.out

```diff
@@ -316,7 +316,11 @@
               id : {α : Type} → α → α @ ⟨34, 35⟩-⟨34, 37⟩
             val : Nat → Nat := id @ ⟨34, 28⟩-⟨34, 31⟩
     pair : A × A := ({ val := id }, { val := id }) @ ⟨34, 2⟩-⟨34, 6⟩
-def id.{u} : {α : Sort u} → α → α :=
-fun {α : Sort u} (a : α) => a
-[Elab.info] command @ ⟨37, 0⟩-⟨37, 9⟩ @ Lean.Elab.Command.elabPrint
-  id : {α : Sort u} → α → α @ ⟨37, 7⟩-⟨37, 9⟩
+def Nat.xor : Nat → Nat → Nat :=
+bitwise bne
+[Elab.info] command @ ⟨37, 0⟩-⟨38, 10⟩ @ Lean.Elab.Command.expandInCmd
```

Comment:
This tree is now printed correctly and exactly once

Reply:
For a while we have been wrapping top-level commands in `ofCommandInfo`, even when they expand to multiple ones as here. Are any cases of multiple top-level info trees left? If not, maybe we can enforce this and get rid of all the annoying [loops](https://github.com/leanprover/lean4/blob/8454102fc4b0b709ecf068a9d6db8e2b3f2a737c/src/Lean/Server/FileWorker/RequestHandling.lean#L91).

Reply:
We can't change `Command.State` since we want it to implement `MonadInfoTree`; at best we could assert there is a single root in `compileNextCmd` and then store that in a new `Snapshot` field

Reply:
That might work, I'll give implementing that in the server a go later.